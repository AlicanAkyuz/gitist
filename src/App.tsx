import { hot } from 'react-hot-loader/root';
import React, { FunctionComponent, Fragment } from 'react';

import Routes from './routes';
import Navbar from './components/Navbar';

const App: FunctionComponent = () => (
  <Fragment>
    <Navbar />
    <Routes />
  </Fragment>
);

export default hot(App);
