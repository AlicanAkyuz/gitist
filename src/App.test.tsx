import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

it('App corrently renders a navbar and Routes component', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
