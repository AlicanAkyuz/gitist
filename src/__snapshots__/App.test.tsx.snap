// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App corrently renders a navbar and Routes component 1`] = `
<Fragment>
  <Navbar />
  <Routes />
</Fragment>
`;
